docs/en/observability/logs-stream.asciidoc

```diff
@@ -1,7 +1,11 @@
 [[logs-stream]]
 = Stream any log file
 
-This guide shows you how to send a log file to Elasticsearch using a standalone {agent} and configure the {agent} and your data streams using the `elastic-agent.yml` file, and query your logs using the data streams you've set up.
+This guide shows you how to manually configure a standalone {agent} to send your log data to {es} using the `elastic-agent.yml` file..
+
+If you don't want to manually configure the {agent}, you can use the **Monitor hosts with {agent}** quickstart. Refer to the <<quickstart-monitor-hosts-with-elastic-agent,quickstart documentation>> for more information.
```

Comment:
I boldly support the splitting of infinitives!

Reply:
✊